### PR TITLE
Add PvP chain support checks, adjust merchandise chain check

### PR DESCRIPTION
**Pull request by Chriserus**
### All Submissions

* [x] Can you post a screenshot of your changes (if applicable)?
image

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

Adds chain based restriction (supports enviroments) for PvP and enhances it for Merchandise

### CODE REVIEW

File: frontend/app-config.json [hunk 1]
```diff
@@ -405,7 +406,8 @@
           "exchangeUrl": "https://app.pangolin.exchange/#/swap?inputCurrency=0x483416eb3afa601b9c6385f63cec0c82b6abf1fb&outputCurrency=0xc7198437980c041c805a1edcba50c1ce5db95118",
           "fightGas": "0.006"
         }
-      }
+      },
+      "pvpSupportedChains": ["HECO","OEC"]
```
Comment (Chriserus) [inline frontend/app-config.json:410]: Let me know which chains will be supported on prod (don't worry, there's still flag in place, so it won't be available, but we can prepare for day 1)

File: frontend/app-config.json [hunk 2]
```diff
@@ -200,7 +200,8 @@
           "exchangeUrl": "",
           "fightGas": "0.006"
         }
-      }
+      },
+      "pvpSupportedChains": ["BSC","HECO","OEC","POLYGON","AVAX"]
```
Comment (Chriserus) [inline frontend/app-config.json:204]: I turned PvP on for all chains for testnet

File: frontend/src/store.ts [hunk 3]
```diff
@@ -743,12 +748,23 @@ export function createStore(web3: Web3) {
...
         if (!currentChain || !merchandiseSupportedChains) {
           state.currentChainSupportsMerchandise = false;
         }
         state.currentChainSupportsMerchandise = merchandiseSupportedChains.includes(currentChain);
       },
 
+      updateCurrentChainSupportsPvP(state: IState) {
+        const env = window.location.href.startsWith('https://test') ? 'test' : 'production';
+        const currentChain = localStorage.getItem('currentChain') || 'BSC';
+        const pvpSupportedChains = config.environments[env].pvpSupportedChains;
```
Comment (kcper) [inline frontend/src/store.ts:761]: I think you can use getConfigValue here, can't you?

**Review by kcper — APPROVED** [review]
Approved

**Pull request merged**